react-components/table-intro/step3.md

```diff
@@ -1,90 +1,14 @@
-Now that you have the basic table structure and imports in place, let's express column and row data as objects. This will allow us to attach and adjust data about individual rows/cells/columns later on.
+With the basic table structure and imports in place, next convert the columns definition into an array of objects.
 
-## Task
+1) <strong>Locate the table columns definition.</strong>
 
-1) Locate the following code just below the last import statement `const columns = ["First column", "Second column", "Third column"];`.
+For example: `const columns = ["First column", "Second column", "Third column"];`
```

Comment:
When you say for example, it makes me think that this could be one of many ways that itll look and I think for the purposes of this tutorial we should be specific and clear about what they should look for. So maybe replace with "It should look like this"